### ISSUE TITLE
Text Selection: Long press + force touch selects words

### ISSUE THREAD

**Issue by matthew-carroll** [flutter/flutter#20770]
Given an editable text field, when the user long presses then a cursor appears at the user's fingertip.  If the user then presses harder, activating force touch, then the entire word is select.  Moving left/right will expand the selected area left/right.

**Comment by jslavitz** [comment]
I'm not sure if the functionality you're describing has changed recently, but I'm seeing a couple of things. Seems like the functionality is as follows: Force tap (very short press with pressure and quick release) -> select word. Force press and hold -> selected words are draggable. The functionality you're talking about in regards to force touching and dragging to change the selection seems like a bug on iOS because I was a) not able to consistently make it work and b) do not think there is an available gesture to describe the functionality.

**Comment by justinmc** [comment]
I agree with @jslavitz and think that we should only implement the "force tap" behavior for now.  I played around with long force touching and dragging and got a lot of inconsistent behavior.
